### ISSUE TITLE
Docs, Support, and Fixes for Directus 6 (Legacy)

### ISSUE THREAD

**Issue by christiaangoossens** [directus/directus#2259]
# Bug Report

While v6 is EOL, most people are still using it in production, as v7 is not quite ready for this. We would however expect some level of support during this period, as well as documentation availability.

## Expected Behavior

- Old docs (https://github.com/directus/api-docs-6-legacy) being available under a special url.
- Old repositories being available for pull-requests for fixes for v6, which would be still supplied to users.

## Actual Behavior

Confusion.

## Other Context & Screenshots

I like you project a lot, but this seems wrong. Can anyone explain the situation?

_EDIT: Comment sounded to harsh, not what was meant._

**Comment by benhaynes** [comment]
Hello @christiaangoossens — I sympathize with your situation, but the reality is: _free and open-source products don't have the resourcing to support what you're asking for. Therefore this burden falls to the community._

We have one full-time staff member (myself, unpaid, front-end only) and two part-time core developers that I _personally_ pay just to keep the project alive. We can not offer any free support for obvious reasons, and paid support is only available for v7 since only one of our devs is even familiar with v6 (and his time is very limited).

The v6 Docs are available on GitHub, if someone from the community wants to generate a fancier web-version of those (eg: on Netlify) that would be awesome and we'll link to them. But markdown gets the job done and we simply don't have any free time for this enhancement unfortunately.

Version 6 was too complex to maintain, had a steep dev learning curve, and was difficult to upgrade. That is why we created version 7 — so that we could resolve all of those previous issues at once instead of letting the entire project die a slow death.

If you feel that Directus 7 isn't production ready, then my opinion is that it's better to invest time getting it stable instead of maintaining a legacy product. We only have time to do one or the other, not both.

Also, in addition to all the Directus work that you see publicly, we're also working behind the scenes on a Cloud service and Marketplace so that this project has the financial means to can stay active long-term. Again, this is all happening with a handful of dev hours per week.

### TLDR: Scope, Time, Cost... you can only choose two. If users need more scope, then they should plan on [donating their Time or Money](https://docs.directus.io/supporting-directus.html) to the project.

**Comment by christiaangoossens** [comment]
Thanks for your response, in my opinion a TLDR is not necessary, everybody should take the time to learn about the product and read this response. I was only asking if the old documentation site could be left up, as well as community made patches for security could still be applied for v6, as I am aware of the nature of the project, but if that is not possible, so be it.

Thank you, and the others of the team, for creating Directus, and maintaining version 7. Good luck with further development! :+1:

**Comment by benhaynes** [comment]
Of course! It's an important point to bring up. The lifespan of a product is an important part of due diligence, so it is important to us to support users of Directus. This was actually the main focus of v7 development and it being versionless — to "rip the bandaid off" with breaking changes now so we could make all future upgrades much easier. From Directus 7 onward it will be easy to stay up to date — and we can support versions much longer.

I'll keep this open for a bit since your question comes up quite a bit over Slack, Intercom, Email, etc.

Thanks for bringing it up!

**Comment by benhaynes** [comment]
I'm going to close this — but I'm leaving it pinned so other users can easily find it!
